src/components/TopBar/TopBar.vue

```diff
@@ -347,6 +347,7 @@ export default {
 		width: 100%;
 		overflow: hidden;
 		min-height: var(--default-clickable-area);
+		font-size: 13px;
```

Comment:
This makes the text smaller than default even when there is only a title without a description.
And there is some additional space if we reduce the padding.

@jancborchardt What do you think is better here?

Also, we can stop showing the description here and show it in the sidebar. It is one line for a long (potentially rich) text anyway.

Reply:
Reverted changes, will provide this suggestion as a separate PR